src/main/frontend/components/editor.cljs

```diff
@@ -450,8 +450,12 @@
 
 (defn get-editor-style-class
   "Get textarea css class according to it's content"
-  [content format]
-  (let [content (if content (str content) "")]
+  [block content format]
+  (let [content (if content (str content) "")
+        heading (-> block :block/properties :heading)
+        heading (if (true? heading)
+                  (min (inc (:block/level block)) 6)
```

Comment:
When auto is enabled I'm seeing an off by one in heading size, so I'm guessing the bug is here.

<img width="352" alt="Screen Shot 2023-02-03 at 12 05 52 PM" src="https://user-images.githubusercontent.com/97210743/216663734-528f879d-26e4-40fd-8141-06bcdb18c28d.png">

These two blocks are that same outline level but the auto heading one looks smaller

Reply:
Is the first one set as an explicit heading level and the second one set to auto? If that's the case, see the related discussion here https://github.com/logseq/logseq/issues/8337.

Reply:
I agreed with @lef-est that it should be consistent since Markdown and Org allow multiple H1 headings anywhere.

Reply:
@sprocketc Yep. First is explicit and second is auto

Reply:
You are allowed to explicitly set the heading you want. One of the reasons I started using auto myself and decided to re-introduce it, is that it adapts depending on the viewing context and almost always generates an accessible structure. The fact that it will makes a nested block an h1 when you click on it to view it in isolation, makes me like this feature even more.